pkg/metastore/metastore_utils.go

```diff
@@ -27,6 +27,17 @@ func ReplaceBranchName(location, branch string) (string, error) {
 	return u.Scheme + "://" + u.Host + "/" + branch + "/" + p.Path, nil
 }
 
+func ReplaceExternalToLakeFSImported(location, repo, branch string) (string, error) {
+	u, err := url.Parse(location)
+	if err != nil {
+		return "", err
+	}
+	if u.Scheme == "" || u.Host == "" {
+		return "", ErrInvalidLocation
```

Comment:
so we will have more information in case it fails:
```suggestion
		return "", fmt.Errorf("%w: %s", ErrInvalidLocation, location)
```